docs/UsersGuide.asciidoc

```diff
@@ -393,12 +393,12 @@ releases, jobs for which long-lasting bug reports exist, etc.
 === Filtering test results and builds ===
 
 At the top of the test results overview page is a form which allows filtering tests by result,
-architecture and TODO-status.
+architecture and TODO-status. TODO-status means that tests still require review.
```

Comment:
```suggestion
architecture and TODO-status. TODO means that tests still require review.
```

Additionally, "review" could be a link leading to the "Review badges" section where the term "review" is clarified. (Maybe there's a more fitting target? I just thing that "still require review" is also a bit vague.)

Reply:
I agree that it is kind of vague, althoug the Review badge section also just escribes "being reviewed" as "a bug reference or a test issue reason is stated"